lib/generators/flipper/templates/migration.rb

```diff
@@ -12,12 +12,10 @@ def self.up
       t.string :value
       t.timestamps null: false
     end
-    add_foreign_key :flipper_gates, :flipper_features, on_delete: :cascade
```

Comment:
I *believe* these were required for Rails 3.2 to function properly... Will go look at that in a bit.